README.md

```diff
@@ -1,14 +1,52 @@
 # Generator used by packit-service
 
-Generator is used for deploying one POD from another POD.
-The aim is to separate `packit-service` and generation sources in order
-to be secure.
+Run untrusted code in a sandbox - an OpenShift pod.
 
-## How to use generator
+
+## Usage
+
+The simplest use case is to invoke a command in a new openshift pod:
```

Comment:
The prerequisite is to run `oc cluster up`. I guess you have to set up also variables like `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT`, haven't you?